angular/lib/src/lib/rating/rating.component.ts

```diff
@@ -30,7 +29,7 @@ export class RatingStarDirective {
 }
 
 @Component({
-	selector: 'au-rating',
+	selector: 'div[auRating]',
```

Comment:
```suggestion
	selector: '[auRating]',
```
I think we should not include the tag name in the selector, so that people can use any tag they want.